From 5e05e3ca202471cf8c62558f6484b9661d0ee558 Mon Sep 17 00:00:00 2001
From: The Android Automerger <android-build@google.com>
Date: Mon, 28 Mar 2016 18:12:02 -0700
Subject: [PATCH 27/55] "MOB30F"

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 5d9fcf9..d212d32 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=MOB30E
+export BUILD_ID=MOB30F
-- 
2.7.4

