From fb4c863ae9dc01791e27c2ff65b55e794c04b116 Mon Sep 17 00:00:00 2001
From: The Android Automerger <android-build@google.com>
Date: Fri, 24 Jun 2016 15:14:36 -0700
Subject: [PATCH 40/55] "M4B30Q"

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index df6c9c4..fb70c8d 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=MOB30P
+export BUILD_ID=M4B30Q
-- 
2.7.4

