From ade055fe3ce7f937ba99009d50e8efb0506f16fc Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Wed, 7 Sep 2016 17:37:40 -0700
Subject: [PATCH 21/63] NBD90X

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 53907a7..5fcbdd8 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD90W
+export BUILD_ID=NBD90X
-- 
2.9.3

