From 9fe1a6cca4c1ad8d7f1578dcf908da399c9273ca Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Mon, 12 Sep 2016 14:12:27 -0700
Subject: [PATCH 23/63] NBD90Z

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index bdc56b4..77fea5a 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD90Y
+export BUILD_ID=NBD90Z
-- 
2.9.3

