From cdd6767cacdc86ad80fd165e98ce238e3e8ca3e4 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Mon, 26 Sep 2016 18:47:35 -0700
Subject: [PATCH 27/63] NBD91C

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index d736387..06d883f 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD91B
+export BUILD_ID=NBD91C
-- 
2.9.3

