From 8ae25e87a026eb3ea9b4ecce0518600fc2d174d4 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Fri, 30 Sep 2016 12:37:26 -0700
Subject: [PATCH 37/63] NBD91J

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 61cf088..0c743fe 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD91I
+export BUILD_ID=NBD91J
-- 
2.9.3

