From 09703e6a255bc85f37b64035392713e805ba2922 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Fri, 30 Sep 2016 14:25:37 -0700
Subject: [PATCH 38/63] NBE92

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 0c743fe..7bb0832 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD91J
+export BUILD_ID=NBE92
-- 
2.9.3

