From 1d0b8bda5cb7c6e69a695c5fd2b90dabe7fe4ce6 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Wed, 26 Oct 2016 11:49:32 -0700
Subject: [PATCH 50/63] NBF26

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 0df382d..9ac255e 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBD91Q
+export BUILD_ID=NBF26
-- 
2.9.3

