From 49c80a376e093c8f62c21922c10edc0f1c6d7677 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Thu, 27 Oct 2016 10:05:51 -0700
Subject: [PATCH 56/63] NBF27

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 9ac255e..d85fca2 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NBF26
+export BUILD_ID=NBF27
-- 
2.9.3

