From 047e02144873ffcfe6387ef151b1a8166305beaf Mon Sep 17 00:00:00 2001
From: The Android Automerger <android-build@google.com>
Date: Thu, 17 Dec 2015 13:15:30 -0800
Subject: [PATCH 13/55] "MMB29U"

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 16709e3..f55b184 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=MMB29T
+export BUILD_ID=MMB29U
-- 
2.7.4

