From 67d338e88aae4479c4314a7006c0cefae3887a06 Mon Sep 17 00:00:00 2001
From: The Android Automerger <android-build@google.com>
Date: Tue, 16 Feb 2016 12:28:52 -0800
Subject: [PATCH 16/55] "MOB29W"

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index e57fdb2..e748a8d 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=MMB29V
+export BUILD_ID=MOB29W
-- 
2.9.3

