From 5317078ea635ae874f9d7bd5a15990a9ebb1d77a Mon Sep 17 00:00:00 2001
From: The Android Automerger <android-build@google.com>
Date: Thu, 26 May 2016 13:53:24 -0700
Subject: [PATCH 36/55] "MOB30N"

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index db41f98..3f45e70 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=MOB30M
+export BUILD_ID=MOB30N
-- 
2.9.3

