From d35611261e7b2a465718052eed34f8946198e144 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Thu, 8 Dec 2016 10:26:51 -0800
Subject: [PATCH 08/27] NMF69

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 2e8dab6..9fffd77 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NMF26T
+export BUILD_ID=NMF69
-- 
2.7.4

