From fbb48f3ce35e1e58d15ab6cfa16d1dc4f3d984d2 Mon Sep 17 00:00:00 2001
From: Adam Seaton <aseaton@google.com>
Date: Thu, 8 Dec 2016 09:37:35 -0800
Subject: [PATCH 09/27] Revert Security String to December

---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 3bda486..2c8a21f 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -114,7 +114,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2017-02-01
+      PLATFORM_SECURITY_PATCH := 2016-12-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

