From 0e4fb3bd348e850080e182226c347ec4e07b5663 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Mon, 7 Nov 2016 16:34:29 -0800
Subject: [PATCH 21/23] NMF26I

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 23bdc1d..f96e259 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NMF26J
+export BUILD_ID=NMF26I
-- 
2.7.4

