From d3bf99ac1dc8cb71d6ddcf41c4d2dc9d526c7a91 Mon Sep 17 00:00:00 2001
From: gitbuildkicker <android-build@google.com>
Date: Fri, 4 Nov 2016 14:26:41 -0700
Subject: [PATCH 26/27] NMF26H

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index f96e259..5ecb669 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=NMF26I
+export BUILD_ID=NMF26H
-- 
2.7.4

